apps/api/.env

```
@@ -0,0 +1,4 @@
+VITE_SUPABASE_URL=https://jbycqmqpqvexthykwuff.supabase.co
+VITE_SUPABASE_ANON_KEY=sb_publishable_mQwG5FSE4mRNANR3OtNnkQ_ohY20jHw
+password=okazumiso370
```

dotenv-linter (4.0.0)

[warning] [LowercaseKey] The password key should be in uppercase

(LowercaseKey)